dlt-database/entity/0001-init_db/InvalidTransaction.ts

```diff
@@ -0,0 +1,10 @@
+import { Entity, PrimaryGeneratedColumn, Column } from 'typeorm'
+
+@Entity('invalid_transactions')
+export class InvalidTransaction {
+  @PrimaryGeneratedColumn('increment', { unsigned: true, type: 'bigint' })
+  id: number
+
+  @Column({ name: 'iota_message_id', type: 'binary', length: 32, nullable: true })
+  iotaMessageId?: Buffer
```

Comment:
Does an entry in this table will not always have an iotaMessageId or could a transaction be invalid without an iotaMessage?
In the first case iotaMessageId could be nullable: false

Reply:
You're right, I'm sorry, the longer the review drags on, the more impatient I get and the more mistakes creep in.